docs/source/administration.rst

```diff
@@ -106,6 +106,13 @@ they represent:
 :contract.py: The `Contract` model which is used to generate the final contract document and other
               support models;
 
+.. image:: _images/sponsors-db.png
```

Comment:
Not sure if the image (and the dependency that generates it) is necessary here as it is quite complex and I feel more lost with it 😓